source/dotnet/wholesale-api/WebApi.UnitTests/Application/IntegrationEventsManagement/IntegrationEventDispatcherServiceTests.cs

```diff
@@ -55,4 +55,34 @@ public class IntegrationEventDispatcherServiceTests
         // Assert
         integrationEventDispatcherMock.Verify(x => x.DispatchIntegrationEventsAsync(It.Is<int>(i => i > 0)));
     }
+
+    [Theory]
+    [AutoMoqData]
+    public async Task DispatchIntegrationEventsAsync_CallsDeleteOlderDispatchedIntegrationEventsOnIntegrationEventCleanUpService(
+        [Frozen] Mock<IIntegrationEventCleanUpService> integrationEventCleanUpServiceMock,
+        IntegrationEventService sut)
+    {
+        // Arrange
+        const int days = 10;
+
+        // Act
+        await sut.DeleteOlderDispatchedIntegrationEventsAsync(days);
+
+        // Assert
+        integrationEventCleanUpServiceMock.Verify(x => x.DeleteOlderDispatchedIntegrationEvents(days));
+    }
+
+    [Theory]
+    [AutoMoqData]
+    public async Task DeleteOlderDispatchedIntegrationEventsAsync_CallsCommitOnUnitOfWork(
```

Comment:
If you prefer a shorter name, here is a suggestion
```suggestion
    public async Task DeleteOlderDispatchedIntegrationEventsAsync_CommitsUnitOfWork(
```